pages/about.html

```diff
@@ -0,0 +1,226 @@
+        <h1 style="color: red; font-weight: bold; font-size: xxx-large;">
+          GLOBAL SUMMIT 2022
+        </h1>
+      </div>
+
+      <div class="d-flex mainPara text-black my-3 justify-content-center">
+        <p style="background: white;">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ad
+          laudantium voluptatum, modi natus perspiciatis. Lorem ipsum dolor sit
+          amet consectetur adipisicing elit. Lorem ipsum dolor sit amet
+          consectetur adipisicing elit. Fugit ad laudantium voluptatum, modi
+          natus perspiciatis. Lorem ipsum dolor sit amet consectetur adipisicing
+          elit.
+        </p>
+      </div>
+      <div class="d-flex m-1 justify-content-center">
+        <h5>
+          Please contact us per email for any further questions about Dev Global
+          Summit 2022!
+        </h5>
+      </div>
+      <div class="d-flex m-7 pb-5 justify-content-center">
+        <h5 style="text-decoration: underline;">enhanceskills2022@gmail.com</h5>
+      </div>
+    </div>
+
+    <!-- // Section 2 // -->
+
+    <div class="container-fluid p-3 bg-white mt-5">
+      <div class="container">
+        <div class="d-flex p-3 justify-content-center">
+          <h3>Enhance Your Skills Global Summit 2020 Logo</h3>
+        </div>
+        <div class="d-flex justify-content-center">
+          <h6 class="textSection3">
+            Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur,
+            adipisicing elit. Excepturi aliquid aperiam debitis accusantium.
+          </h6>
+        </div>
+        <div class="d-flex p-5 m-3 justify-content-center">
+          <img src="../images/Enha (1).png" alt="" />
+        </div>
+      </div>
+    </div>
+    <hr />
+
+    <!-- // Section 3 //  -->
+
+    <div class="container-fluid">
+      <div class="container">
+        <div class="d-flex p-3 justify-content-center">
+          <h3 style="text-align: center;">
+            Enhance Your Skills Global Summit 2020 Logo
+          </h3>
+        </div>
+        <div class="d-flex justify-content-center">
+          <h6 class="textSection3">
+            Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur,
+            adipisicing elit. Excepturi aliquid aperiam debitis accusantium.
+          </h6>
+        </div>
+        <div class="row data-center">
+          <div class="col-md-6">
+            <div class="imgPast1">
+              <h1>2015</h1>
+              <h3 style="text-align: center;">
+                Enhance Your Skills Global Summit in Vince
+              </h3>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="imgPast2">
+              <h1>2019</h1>
+              <h3 style="text-align: center;">
+                Enhance Your Skills Global Summit in Buenos Aires
+              </h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly seperate the two articles so as to match the given design.

Your design
![image](https://user-images.githubusercontent.com/81932478/205465172-a044150f-4cd1-41e7-9fde-acec07a8781d.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205465194-c42c29aa-54a1-41b8-b49f-fc8061be17b6.png)

Reply:
Thanks for the suggestion. The design is working fine on the given screen sizes. Both for mobile and desktop.